automation-tests/test/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
should move this to the top level of the package and have it extend the common tsconfig file like we do in the other packages.  moduleResolution is in there twice.  This will ensure we are able to easily upgrade typscript & node without having to do it for this package and the other separately.